modules/cluster-logging-about-elasticsearch.adoc

```diff
@@ -3,14 +3,16 @@
 // * logging/cluster-logging.adoc
 
 [id="cluster-logging-about-elasticsearch_{context}"]
-= About the logstore 
+= About the log store 
 
 {product-title} uses link:https://www.elastic.co/products/elasticsearch[Elasticsearch (ES)] to organize the log data from Fluentd into datastores, or _indices_. 
 
 Elasticsearch subdivides each index into multiple pieces called _shards_, which it spreads across a set of Elasticsearch nodes in an Elasticsearch cluster.
 You can configure Elasticsearch to make copies of the shards, called _replicas_. Elasticsearch also spreads these replicas across
 the Elasticsearch nodes. The *ClusterLogging* Custom Resource (CR) allows you to specify the replication policy to provide data redundancy and resilience to failure. You can also define how long indices should be retained using a retention policy in the Cluster Logging CR.
 
+The cluster logging Elasticsearch instance is optimized and tested for short term storage, approximately seven days. If you want to retain your logs over a longer term, it is recommended you move the data to a third-party storage system. 
```

Comment:
Fine as it is, just perhaps to consider:

* Maybe "short term storage of approximately seven days"
* recommended that you